en/publish/publish-native.md

```diff
@@ -77,7 +77,11 @@ Encrypt the published script. After build, the JSC file is generated in the `src
 
 **Zip Compress**: You can reduce the size of your scripts by checking them
 
-![](publish-native/js_secret.PNG)
+![](publish-native/js_secret.png)
+
+### Build Scripts Only
+
+When you rebuild the project, if you only modify the script, checking this option will only rebuild the script, which can greatly reduce the build time.
```

Comment:
When you build the project, if you only modified some scripts, checking this option will only rebuild the scripts, which can greatly reduce the build time.